engine/security/src/main/scala/pl/touk/nussknacker/ui/security/oauth2/OpenIdConnectProfile.scala

```diff
@@ -0,0 +1,56 @@
+package pl.touk.nussknacker.ui.security.oauth2
+
+import io.circe.generic.JsonCodec
+import io.circe.generic.extras.{Configuration, ConfiguredJsonCodec, JsonKey}
+import pl.touk.nussknacker.ui.security.api.{LoggedUser, RulesSet}
+import pl.touk.nussknacker.ui.security.oauth2.OAuth2Profile.getUserRoles
+
+import scala.concurrent.duration.Deadline
+
+@ConfiguredJsonCodec case class OpenIdConnectUserInfo
+(
+  @JsonKey("sub") subject: Some[String],
+
+  name: Option[String],
+  @JsonKey("given_name") givenName: Option[String],
+  @JsonKey("family_name") familyName: Option[String],
+  @JsonKey("middle_name") middleName: Option[String],
+  nickname: Option[String],
+  @JsonKey("preferred_username") preferredUsername: Option[String],
+  profile: Option[String],
+  picture: Option[String],
+  website: Option[String],
+  email: Option[String],
+  @JsonKey("email_verified") emailVerified: Option[Boolean],
+  gender: Option[String],
+  @JsonKey("birth_date") birthDate: Option[String], //ISO 8601
```

Comment:
If we know it's ISO maybe we can just use decoded version?

Reply:
Replaced with `LocalDate`